Aquifer/IO.swift

```diff
@@ -12,13 +12,13 @@ import Foundation
 import Swiftz
 
 /// Returns a `Pipe` that reads input from `stdin` line-by-line and terminates on end-of-input.
-public func stdinLn<UO, UI>() -> Proxy<UO, UI, (), String, ()> {
+public func stdinLn() -> Producer<String, ()>.T {
```

Comment:
It's been a while since I tried…maybe I was just running into a compiler bug or something.